### PR TITLE
Firmware V2 for the hardware probes

### PR DESCRIPTION
**Pull request by kernelgurumeditation**
Don't merge it YET! ->STILL UNDER TESTS <-

includes:
Fix for the SSH host keys changing at every boot
Automatic container update feature
New user "devlogs" to enable checking for firmware logs

### CODE REVIEW

File: meta-jsdelivr/recipes-jsdelivr/jsdelivr-configure/files/sshd_config_readonly_jsdelivr [hunk 1]
```diff
@@ -124,5 +124,10 @@ Match User logs
   AllowTCPForwarding no
   ForceCommand /bin/bash -c "/usr/bin/docker logs globalping-probe -f "
 
+Match User devlogs
```
Comment (jimaek) [inline meta-jsdelivr/recipes-jsdelivr/jsdelivr-configure/files/sshd_config_readonly_jsdelivr:127]: lets document in readme how this user works and what is used for

Comment (kernelgurumeditation) [reply meta-jsdelivr/recipes-jsdelivr/jsdelivr-configure/files/sshd_config_readonly_jsdelivr:127]: Sure, I'm just waiting for the probe to finish the tests(should finish in 3/4 days)

**Comment by MartinKolarik** [comment]
> If, for any reason, the container fails to start (corruption in the file/card or  bug in the container), 
the jsdelivr-systemWatchdog.sh script  will reset the system to use the bundled container.

Let's say the container updates successfully 10 times over two years. Then 11th update fails. Does it return to the 10th (previously working) version or to the very first, two years old version?

**Comment by kernelgurumeditation** [comment]
> > If, for any reason, the container fails to start (corruption in the file/card or  bug in the container),
> > the jsdelivr-systemWatchdog.sh script  will reset the system to use the bundled container.
> 
> Let's say the container updates successfully 10 times over two years. Then 11th update fails. Does it return to the 10th (previously working) version or to the very first, two years old version?

**Comment by kernelgurumeditation** [comment]
Closing this,
Given the changes in the container hub, I need to redo a portion of the system to handle Skopeo limitations when run on an embedded system.....

**Pull request closed without merging**